apps/www/src/components/repositories-list-container.component.tsx

```
@@ -0,0 +1,267 @@
+        totalItems: 0,
+        totalPages: 0,
+        currentPage: 1,
+        itemsPerPage: repositoriesPerPage,
+        hasNextPage: false,
+        hasPreviousPage: false
+      }
+    )
+  }, [data?.pagination, repositoriesPerPage])
+
+  return (
+    <div className='flex w-full flex-col gap-4' role='main' aria-label='Lista de repositórios do GitHub'>
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<main>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)